### PR TITLE
[DOCS] Edit Fleet operation summaries

### PR DESCRIPTION
**Pull request by lcawl**
Relates to https://github.com/elastic/elasticsearch-specification/issues/3226

This PR edits the Fleet summaries in https://www.elastic.co/docs/api/doc/elasticsearch/group/endpoint-fleet based on content from https://www.elastic.co/guide/en/elasticsearch/reference/current/fleet-apis.html

### CODE REVIEW

File: specification/fleet/global_checkpoints/GlobalCheckpointsRequest.ts [hunk 1]
```diff
@@ -23,6 +23,9 @@ import { Duration } from '@_types/Time'
 import { Checkpoint } from '../_types/Checkpoints'
 
 /**
+ * Get global checkpoints.
+ * Get the current global checkpoints for an index.
+ * This API is design for internal use by the Fleet server project.
```
Comment (marciw) [inline specification/fleet/global_checkpoints/GlobalCheckpointsRequest.ts:28]:
```suggestion
 * This API is designed for internal use by the Fleet server project.
```

File: specification/fleet/search/SearchRequest.ts [hunk 2]
```diff
@@ -52,8 +52,9 @@ import { Duration } from '@_types/Time'
 import { Checkpoint } from '../_types/Checkpoints'
 
 /**
- * The purpose of the fleet search api is to provide a search api where the search will only be executed
- * after provided checkpoint has been processed and is visible for searches inside of Elasticsearch.
+ * Run a Fleet search.
+ * The purpose of the Fleet search API is to provide a search api where the search will be run only
```
Comment (marciw) [inline specification/fleet/search/SearchRequest.ts:56]:
```suggestion
 * The purpose of the Fleet search API is to provide a search API where the search will be run only
```

Comment (marciw) [reply specification/fleet/search/SearchRequest.ts:56]: actually maybe just "provide an API where..." ?

**Review by marciw — APPROVED** [review]
LGTM w/ a couple of tiny typos

File: specification/fleet/search/SearchRequest.ts (hunk 2 above)
Comment (lcawl) [reply specification/fleet/search/SearchRequest.ts]: Thanks! Updated in 2934aade66cf6f0ab83c78699de166ce675bc2db

**Pull request merged**